scripts/pushArgos.js

```diff
@@ -0,0 +1,61 @@
+/* eslint-disable no-console */
+const util = require('util');
+const glob = require('fast-glob');
+const fse = require('fs-extra');
+const chunk = require('lodash/chunk');
+const childProcess = require('child_process');
+
+const execFileAsync = util.promisify(childProcess.execFile);
+
+async function exec(command, args) {
+  const options = {
+    cwd: process.cwd(),
+    env: process.env,
+    stdio: 'pipe',
+    encoding: 'utf-8',
+  };
+
+  const results = await execFileAsync(command, args, options);
+  return results.stdout;
+}
+
+const screenshotsBase = 'test/regressions/screenshots/chrome';
+const BATCH_COUNT = 2;
+
+async function run() {
+  let screenshots = await glob(`${screenshotsBase}/**/*`);
+  screenshots = chunk(screenshots, Math.ceil(screenshots.length / BATCH_COUNT));
+
+  await Promise.all(
+    screenshots.map((chunks, chunkIndex) =>
```

Comment:
At this point `screenshots` contains the `chunks`. But each item is a single `chunk`.